bounties_api/std_bounties/management/commands/bounties_subscriber.py

```diff
@@ -46,18 +46,23 @@ def handle(self, *args, **options):
                 message = Message.from_event(messages[0])
 
                 # If someone uploads a data hash that is faulty, then we want to blacklist all events around that
-                # bounty id. We manage this manually
-                blacklist_exists = redis_client.exists('blacklist:{}'.format(message.bounty_id))
-                logger.info('trying blacklist:{}'.format(message.bounty_id))
-                logger.info('got: {}'.format(blacklist_exists))
+                # bounty id. It can either be a permanent blacklist, typically added manually, or a pending blacklist.
+                # All the events in the pending blacklist will retry later.
+                permanent_blacklist = redis_client.get('blacklist:{}'.format(message.bounty_id))
```

Comment:
Changed this to a `get` in case we want to toggle some of these on/off in the future by setting the boolean value in the key. Kept `exists` for the list, as that will only return true for as long as there are entries in the list.